Remove backup resources (#8261)
kyma-project/kyma: docs/kyma/04-10-upgrade.md

2. Upgrade from version 1.1 to version 1.2.
3. Upgrade from version 1.2 to version 1.3.

> >**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by taking a backup from your cluster before upgrade. Read [this](/components/backup/#velero-tutorial) document to learn more about backups.

Reviewer: Please change link to tutorial to `/components/backup/#tutorials-taking-backup-using-velero`. You can see how links will look in website from preview of your PR -> https://deploy-preview-8261--kyma-project-docs-preview.netlify.app/components/backup/#tutorials-taking-backup-using-velero We have a concatenation of group of docs (in this case `tutorials`) and title of md :)

Author: Done, thanks :)

> >**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by taking a backup from your cluster before upgrade. Read [this](/components/backup/#tutorials-taking-backup-using-velero) document to learn more about backups.

Reviewer 2: ```suggestion
>**NOTE:** Kyma does not support a dedicated downgrade procedure. You can achieve a similar result by creating a backup of your cluster before upgrade. Read [this](/components/backup/#tutorials-taking-backup-using-velero) document to learn more about backups.
```
